app/src/main/java/com/willowtree/vocable/facetracking/FaceTrackingViewModel.kt

```diff
@@ -1,15 +1,19 @@
 package com.willowtree.vocable.facetracking
 
+import android.content.Context
 import android.content.SharedPreferences
 import androidx.lifecycle.LiveData
 import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
 import com.google.ar.core.AugmentedFace
 import com.google.ar.sceneform.math.Vector3
+import com.willowtree.vocable.R
 import com.willowtree.vocable.utils.VocableSharedPreferences
 import kotlinx.coroutines.*
 import org.koin.core.KoinComponent
+import org.koin.core.get
 import org.koin.core.inject
+import java.util.*
```

Comment:
That's a bunch of imports. But that's pretty typical for android, right?

Reply:
ahh i forgot to push cmd+option+L, i'll do that